src/test/java/examples/ExamplesTest.java

```diff
@@ -0,0 +1,18 @@
+package examples;
```

Comment:
Please add an example feature file in this folder

Reply:
That's already available 
file name is 'users.feature'